src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -623,27 +579,32 @@ private static void BuildPatchesInDrawingRegions(PatchMap map, Random random)
 
     private static void ConnectPatchesBetweenRegions(PatchMap map, Random random)
     {
-        foreach (var region in map.NormallyDrawnRegions)
+        foreach (var region in map.Regions)
         {
             ConnectPatchesBetweenRegions(region.Value, random);
         }
     }
 
     /// <summary>
-    ///   Returns a predefined patch with default values
+    ///   Returns a predefined patch with default values.
     /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Note that after calling this, region and patch has already been linked together.
+    ///   </para>
+    /// </remarks>
     /// <param name="biome">The requested biome</param>
     /// <param name="id">ID of this patch</param>
     /// <param name="region">Region this patch belongs to</param>
-    /// <param name="regionName">Name of the region</param>
+    /// <param name="patchName">Name of the region</param>
```

Comment:
Actually looking at the results this *is* the region name as the patches in a region share the region name. I think instead of doing the regionName to patchName rename, PatchMapNameGenerator should be updated so that `Next`'s documentation says that it returns a region name (used as the base part of patch names in a region).